### DISCUSSION TITLE
Is bulk text manipulation operations on alias descriptions possible?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [anonaddy/anonaddy#481] [Q&A] [answered]
Currently, most of the alias descriptions I have for a few hundred alias are formatted like:

> ~~~MA
> @status #inactive @origin https://forums.ageofempires.com/u/rokejulianlo/preferences/email
> ~~~

However, I want to convert them to:

<blockQuote>

```pandoc
`{status: "inactive"; origin: "https://forums.ageofempires.com/u/rokejulianlo/preferences/email"}`{.json} <!-- https://stackoverflow.com/a/25058886/9731176 -->
```

</blockQuote>

However, I don't see any way to perform such an action. Considering that AnonAddy doesn't have anything like Bitwarden's CLI client (to my knowledge) this doesn't seem possible. Is it?

**Comment by willbrowningme** [comment, marked as answer]
I'm afraid not. I could add an endpoint to allow bulk editing alias descriptions (max 25) however this would only allow you to set them all to the same value.

**Reply by roke-julian-lockhart** [reply]
@willbrowningme, thanks. That would be an acceptable solution. I've filed https://github.com/anonaddy/anonaddy/issues/510#issue-1867275296.
